service/arm_templates/security_groups_setup.json

```diff
@@ -67,27 +70,13 @@
               "protocol": "*",
               "sourcePortRange": "*",
               "destinationPortRange": "*",
-              "sourceAddressPrefix": "[parameters('masterSubnetCidr')]",
+              "sourceAddressPrefix": "*",
               "destinationAddressPrefix": "*",
               "access": "Allow",
               "direction": "Outbound",
               "priority": "4095"
             }
           },
-          {
```

Comment:
this rule does nothing as internal traffic (within subnet) will not hit security groups.